### PR TITLE
Fix/233  make some builders not required

### PR DESCRIPTION
**Pull request by inkhey**
related to #233

### CODE REVIEW

File: preview_generator/preview/builder/document__drawio.py [hunk 1]
```diff
@@ -15,12 +13,20 @@
 from preview_generator.utils import MimetypeMapping
 from preview_generator.utils import executable_is_available
 
+xvfbwrapper_installed = True
+try:
+    from xvfbwrapper import Xvfb
+except ImportError:
+    xvfbwrapper_installed = False
```
Comment (raphj) [inline preview_generator/preview/builder/document__drawio.py:20]: :+1:

File: preview_generator/preview/builder/image__cairosvg.py [hunk 2]
```diff
@@ -3,12 +3,18 @@
 import tempfile
 import typing
 
-import cairosvg
-
+# HACK - G.M - 2020-12-26 - Hack to allow load of module without cairosvg installed
```
Comment (raphj) [inline preview_generator/preview/builder/image__cairosvg.py:6]:
```suggestion
# HACK - G.M - 2020-12-26 - Hack to allow loading modules without cairosvg installed
```

**Review by raphj — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -60,6 +66,9 @@ jobs:
     - stage: tests
       python: "3.8"
     - stage: tests
+      install:
+        # remove 3D support as vtk doesn't support 3.9 yet
+        - pip install ".[cairosvg,video,drawio,scribus,testing]"
```
Comment (inkhey) [inline .travis.yml:71]:
```suggestion
        - pip install ".[all,testing]"
```

Comment (inkhey) [inline .travis.yml:71]:
```suggestion
```

File: README.rst [hunk 4]
```diff
@@ -69,25 +69,128 @@ Video format
...
 
-To handle previews for office documents you will need ``LibreOffice``, if you don't have it already::
+install preview_generator without external addons:
 
-  apt-get install libreoffice
+```bash
+pip install preview-generator
+```
 
+To install alls previews builders dependencies:
```
Comment (grignards) [inline README.rst:88]:
```suggestion
To install all previews builders dependencies:
```

File: README.rst [hunk 5]
```diff
@@ -69,25 +69,128 @@ Video format
...
+```bash
+pip install preview-generator[all]
+sudo apt-get install libreoffice inkscape ufraw-batch ffmpeg xvfb
+DRAWIO_VERSION="12.6.5" && curl -LO https://github.com/jgraph/drawio-desktop/releases/download/v${DRAWIO_VERSION}/draw.io-amd64-${DRAWIO_VERSION}.deb && sudo dpkg -i draw.io-amd64-${DRAWIO_VERSION}.deb
+```
 
 To check dependencies, you can run::
 
+# TODO !!! VERIFY THIS
```
Comment (grignards) [inline README.rst:97]: Did you check?

File: README.rst [hunk 6]
```diff
@@ -69,25 +69,128 @@ Video format
...
+
+HEIC support
+~~~~~~~~~~~~
+
+Building ImageMagick with heic support: `Building ImageMagick with heic support`_ .
+
+.. _`Building ImageMagick with heic support`: doc/build_im_with_heic_support.rst
+
+
+## TODO : Check required ?
```
Comment (grignards) [inline README.rst:189]: ? I think this part is not finished.

File: setup.cfg [hunk 7]
```diff
@@ -9,7 +9,7 @@ force_grid_wrap = 0
 use_parentheses = true
 skip_glob = .eggs*,.venv
 extra_standard_library = hashlib, logging, mimetype, typing, os, sys, abc, io, json, datetime, subprocess, tempfile, shutil, argparse
-known_third_party =PIL,PyPDF2,cairosvg,ffmpeg,filelock,magic,pdf2image,pyexifinfo,pytablewriter,pytest,scribus,wand,xvfbwrapper
+known_third_party =PIL,PyPDF2,filelock,magic,pdf2image,pyexifinfo,pytablewriter,pytest,scribus,wand
```
Comment (grignards) [inline setup.cfg:12]: should this value change? Even if those dependencies are now optional they are still third party.

**Review by grignards — CHANGES_REQUESTED** [review]

File: README.rst (hunk 4 above)
Comment (inkhey) [reply README.rst]: done.

File: README.rst (hunk 5 above)
Comment (inkhey) [reply README.rst]: done.

File: README.rst (hunk 6 above)
Comment (inkhey) [reply README.rst]: done.

File: setup.cfg (hunk 7 above)
Comment (inkhey) [reply setup.cfg]: done by removing this,  isort>5 doesn't need anymore theses informations.

**Review by grignards — APPROVED** [review]

**Pull request merged**